### ISSUE TITLE
Skipping recipe /bioconda-recipes...

### ISSUE THREAD

**Issue by dkoppstein** [bioconda/recipes#142]
Hi, I'm a bit new to Docker, I wanted to update the igblast recipe but when I ran the test set in Docker using the command in the README I got: 

```
sudo docker run -v `pwd`:/bioconda-recipes bioconda/bioconda-builder --packages igblast
Skipping recipe /bioconda-recipes/recipes/igblast

----------------------------------------------------------------------
Ran 0 tests in 1.288s

OK
```

What am I doing wrong?

**Comment by daler** [comment]
It's being [filtered out by build-packages.py](https://github.com/bioconda/recipes/blob/master/scripts/build-packages.py#L65), which is is actually independent of docker. That script is doing a check against the recipe name, which has a [specific format](http://conda.pydata.org/docs/building/pkg-name-conv.html), to see if it is already built.

My guess is that your updated recipe isn't changing that conda recipe name. If the version of `igblast` has changed, then update that and it should trigger a build. Otherwise, you can increment the [build number](http://conda.pydata.org/docs/building/meta-yaml.html#build-number-and-string), which is 0 by default, to trigger a new build for the same version.

TL;DR: either increment the version of `igblast`, or add this to `recipes/igblast/meta.yaml`:

```
build:
  number: 1
```

**Comment by dkoppstein** [comment]
OK, I incremented the build number to 1 and it worked -- thanks!
